bbconf/modules/bedfiles.py

```diff
@@ -755,16 +1019,19 @@ def upload_pephub(self, identifier: str, metadata: dict, overwrite: bool = False
         )
 
     def update_pephub(self, identifier: str, metadata: dict, overwrite: bool = False):
-        if not metadata:
-            _LOGGER.warning("No metadata provided. Skipping pephub upload..")
-            return False
-        self._config.phc.sample.update(
-            namespace=self._config.config.phc.namespace,
-            name=self._config.config.phc.name,
-            tag=self._config.config.phc.tag,
-            sample_name=identifier,
-            sample_dict=metadata,
-        )
+        try:
+            if not metadata:
+                _LOGGER.warning("No metadata provided. Skipping pephub upload..")
+                return False
+            self._config.phc.sample.update(
+                namespace=self._config.config.phc.namespace,
+                name=self._config.config.phc.name,
+                tag=self._config.config.phc.tag,
+                sample_name=identifier,
+                sample_dict=metadata,
+            )
+        except ResponseError as e:
+            _LOGGER.warning(f"Could not update pephub. Error: {e}")
```

Comment:
Should we also be returning False here after the exception? to be consistent with line 1025. Also, shouldn't the return type of the func be a bool?

Reply:
Fixed